other/prevent-naked-pods/03-check.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- command: sleep 5
+- script: if [ $(kubectl get pods -n prevent-naked-pods-ns | grep gooddeployment01 | wc -l) -gt 0 ]; then exit 0; else exit 1; fi
```

Comment:
I thought you could assert on a partial name for a Pod, is that not so? You could just assert on the name `gooddeployment01-` (note the trailing dash character), if it does work, and it would be simpler. Otherwise, ignore this comment.

Reply:
Tried this, didn't work. I wish it did though.